### ISSUE TITLE
How to disable hover tips for python? 

### ISSUE THREAD

**Issue by vsfeedback** [microsoft/PTVS#7391]
_This issue has been moved from [a ticket on Developer Community](https://developercommunity.visualstudio.com/t/How-to-disable-hover-tips-for-python-/10261933)._

---
[severity:I'm unable to use this version]
Hi there

I'm using Python and have a serious problem when debugging:
When I hover over a function call, VS calls that function to show the function result. This leads to a side effect, e.g. if a file is created within that function.
How can I disable or configure the Quick Info? 
TIA
Kind regards
Elmar

---
#### Feedback Bot on 1/27/2023, 07:07 PM: 

(private comment, text removed)

---
### Original Solutions
(no solutions)

**Comment by StellaHuang95** [comment]
Hi there, I couldn't repro this. Are you saying that the function will be called even by only hovering over it? Could you provide repro steps and reopen this ticket so that we can further investigate?

**Comment by ElmarZoeller** [comment]
Hello
I have attached a video (zip archive) which shows that every time I move the mouse over the top line of code in the Python window in the background, a log text is written to the Notepad++ window.
I have checked this problem in VS2019 and VS2022. It is reproducible for me in both versions!
Regards
Elmar
[2023-03-03_15h35_04.zip](https://github.com/microsoft/PTVS/files/11881111/2023-03-03_15h35_04.zip)
